src/celeritas/em/data/UrbanMscData.hh

```diff
@@ -110,13 +108,47 @@ struct UrbanMscIds
 
 //---------------------------------------------------------------------------//
 /*!
- * Device data for step limitation algorithms and angular scattering.
+ * Particle- and material-dependent data for MSC.
+ *
+ * The scaled Zeff parameters are:
+ *
+ *   Particle | a    | b
+ *   -------- | ---- | ----
+ *   electron | 0.87 | 2/3
+ *   positron | 0.7  | 1/2
+ */
+struct UrbanMscParMatData
+{
+    XsGridData xs;  //!< For calculating MFP
+    real_type scaled_zeff{};  //!< a * Z^b
+
+    //! Whether the data is assigned
+    explicit CELER_FUNCTION operator bool() const
+    {
+        return xs && scaled_zeff > 0;
+    }
+};
+
+//---------------------------------------------------------------------------//
+/*!
+ * Device data for Urban MSC.
+ *
+ * This model applies only to electrons and positrons, so the ParticleItems are
```

Comment:
```suggestion
 * This data applies only to electrons and positrons, so the ParticleItems are
```
as the model will be also applicable to muons and hadrons when extended.

Reply:
Good to know! Do anything but the cross sections change?

Reply:
There are a couple of branches for muons and hadrons in msc codes - for an example, d_over_R is different between EM and others.  We put TODOs where relevant branches are needed so that we can put additional lines of codes when migrating to the full simulation.